packages/spacecat-shared-rum-api-client/src/functions/traffic-acquisition.js

```diff
@@ -50,23 +44,26 @@ function transformFormat(trafficSources) {
     owned: value.owned,
     paid: value.paid,
     sources: Object.entries(value)
-      .filter(([source]) => !MAIN_TYPES.includes(source))
+      .filter(([source]) => !MAIN_TYPES.includes(source) && source !== 'channels')
       .map(([source, views]) => ({ type: source, views })),
+    channels: value.channels,
   }));
 }
 
 function handler(bundles) {
   const trafficSources = bundles
-    .map(extractHints)
+    .map(extractTrafficHints)
     .map((row) => {
       const {
         type,
         category,
+        channel,
```

Comment:
the term we use in all the other places related to RUM is `vendor` not `channel`. We should keep the terminology same across all the products

Reply:
I've changed all the internal things to `vendor` from `channel`, but I still kept the `referrer` in the opportunities object